packages/api/test/e2e/queries.e2e.ts

```diff
@@ -136,7 +136,7 @@ describe('e2e queries', () => {
   describe('Staking account derived query', () => {
     it("Get staking account details", async done => {
       const stashId = '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY'; // alice_stash
-      const stakingAccount = await api.derive.staking.stakingAccountDetails(stashId);
+      const stakingAccount = await api.derive.staking.accountDetails(stashId);
```

Comment:
best to call it `accountInfo` if the type is `StakingAccountInfo` otherwise make the type `StakingAccountDetails`

Reply:
done